monkestation/code/datums/stamina_container.dm

```diff
@@ -1,7 +1,8 @@
 /datum/stamina_container
 	///Daddy?
 	var/mob/living/parent
-	///The maximum amount of stamina this container has
+	/// The maximum amount of stamina this container has.
+	/// Don't touch this directly, it is set using set_maximum().
```

Comment:
could make it be protected/private

Reply:
it's fine to read from it - just not to _set_ it - and protected/private also complain if something else reads it